templates/bot/ts/m365/package.json

```diff
@@ -7,11 +7,11 @@
   "main": "./lib/index.js",
   "scripts": {
     "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
-    "dev": "nodemon --watch ./src --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./index.ts",
```

Comment:
This change seems due to the hotfix on main branch... So no need to change in this PR since main will be merged back later.